stdlib/time/src/main/ballerina/time/timelib.bal

```diff
@@ -18,141 +18,149 @@ public const TIME_FORMAT_RFC_1123 = "RFC_1123";
+# + return - The month-of-year, from 1 (January) to 12 (December)
+public extern function getMonth(Time time) returns int;
+
+# Returns the date representation of the given time.
+#
+# + time - The Time record to get the date representation from
+# + return - The day-of-month, from 1 to 31
+public extern function getDay(Time time) returns int;
+
+# Returns the weekday representation of the given time.
+#
+# + time - The Time record to get the weekday representation from
+# + return - The weekday representation from SUNDAY to SATURDAY
+public extern function getWeekday(Time time) returns string;
+
+# Returns the hour representation of the given time.
+#
+# + time - The Time record to get the  hour representation from
+# + return - The hour-of-day, from 0 to 23
+public extern function getHour(Time time) returns int;
+
+# Returns the minute representation of the given time.
+#
+# + time - The Time record to get the  minute representation from
+# + return - The minute-of-hour to represent, from 0 to 59
+public extern function getMinute(Time time) returns int;
+
+# Returns the second representation of the given time.
+#
+# + time - The Time record to get the second representation from
+# + return - The second-of-minute, from 0 to 59
+public extern function getSecond(Time time) returns int;
+
+# Returns the millisecond representation of the given time.
+#
+# + time - The Time record to get the millisecond representation from
+# + return - The milli-of-second, from 0 to 999
+public extern function getMilliSecond(Time time) returns int;
+
+# Returns the date representation of the given time.
+#
+# + time - The Time record to get the date representation from
+# + return - The year representation.
+#            The month-of-year, from 1 (January) to 12 (December).
+#            The day-of-month, from 1 to 31.
+public extern function getDate(Time time) returns (int, int, int);
+
+# Returns the time representation of the given time.
+#
+# + time - The Time record
+# + return - The hour-of-day, from 0 to 23.
+#            The minute-of-hour to represent, from 0 to 59.
+#            The second-of-minute, from 0 to 59.
+#            The milli-of-second, from 0 to 999.
+public extern function getTime(Time time) returns (int, int, int, int);
+
+# Add specified durations to the given time value.
+#
+# + time - The Time record to add the duration to
+# + years - The year representation
+# + months - The month-of-year to represent, from 1 (January) to 12 (December)
+# + days - The day-of-month to represent, from 1 to 31
+# + hours - The hour-of-day to represent, from 0 to 23
+# + minutes - The minute-of-hour to represent, from 0 to 59
+# + seconds - The second-of-minute to represent, from 0 to 59
+# + milliSeconds - The milli-of-second to represent, from 0 to 999
+# + return - Time object containing time and zone information after the addition
+public extern function addDuration(Time time, int years, int months, int days, int hours, int minutes, int seconds,
```

Comment:
Do we need to return Time? Check the possibility of modifying the same time reference passed in to the method.